ext/standard/info.c

```diff
@@ -766,7 +766,7 @@ PHPAPI void php_print_info_htmlhead(void)
 	php_info_print("<html xmlns=\"http://www.w3.org/1999/xhtml\">");
 	php_info_print("<head>\n");
 	php_info_print_style();
-	php_info_print("<title>phpinfo()</title>");
+	php_info_print("<title>PHP %s - phpinfo()</title>", PHP_VERSION);
```

Comment:
The `php_info_print` function is only used to output plain strings. For formatted strings, the `php_info_printf` function must be used.